repositories/common_interfaces.BUILD.bazel

```diff
@@ -160,6 +160,97 @@ py_library(
     ],
 )
 
+ros2_interface_library(
+    name = "shape_msgs",
+    srcs = glob([
+        "shape_msgs/msg/*.msg",
+    ]),
+    visibility = ["//visibility:public"],
+    deps = [
+        ":geometry_msgs",
+    ],
+)
+
+cpp_ros2_interface_library(
+    name = "_cpp_shape_msgs",
+    visibility = ["//visibility:private"],
```

Comment:
Here and down below: the default visibility is private, so, this qualifier can be removed for all targets with names beginning with an underscore.